client/Cargo.toml

```diff
@@ -1,10 +1,25 @@
 [workspace]
-members = [
-    "hwctl",
-    "hwlib"
-]
+members = ["hwctl", "hwlib"]
 resolver = "2"
-exclude = ["rust-vendor",]
+exclude = ["rust-vendor"]
```

Comment:
We should also exclude the debian folder (I forgot to move it from hwlib/Cargo.toml